### PR TITLE
chore: Create yearly archives and cleanup

### PR DESCRIPTION
**Pull request by wilsonmsalberto**
Some lines were from last year and others were from this year.
With an archive, it feels tidier.
Let me know what you think.

### CODE REVIEW

**Comment by BenAtEventbrite** [comment]
Hi @wilsonmsalberto! Thanks so much for taking the time to put this all together!

So I thought about this when I first put this together, but I really wanted to avoid having to maintain the list frequently. What you have is great _now_, but a week from now there should already be conferences that need to be moved over to `2018`. I don't have the time to keep it up-to-date.

Also I really like having the full list of conferences in one place even if they are past ones. It actually makes it easier to see that some conferences need to be updated for 2018. For instance, you moved dotJS to `2017.md` and since it hasn't happened in 2018, it's basically disappeared. I would easily forget to know to update for the 2018 information.

I do like the idea of the 2017 archive for the case that some conferences didn't return in 2018.

What about keeping the README as-is, but also having the yearly archives as you've done?

**Comment by benmvp** [comment]
Hi @wilsonmsalberto! Thanks so much for taking the time to put this all together!

So I thought about this when I first put this together, but I really wanted to avoid having to maintain the list frequently. What you have is great now, but a week from now there should already be conferences that need to be moved over to 2018. I don't have the time to keep it up-to-date.

Also I really like having the full list of conferences in one place even if they are past ones. It actually makes it easier to see that some conferences need to be updated for 2018. For instance, you moved dotJS to 2017.md and since it hasn't happened in 2018, it's basically disappeared. I would easily forget to know to update for the 2018 information.

I do like the idea of the 2017 archive for the case that some conferences didn't return in 2018.

What about keeping the README as-is, but also having the yearly archives as you've done?

**Comment by wilsonmsalberto** [comment]
One of the reasons I was creating the archives was to avoid having conferences that have already passed in the main list but I see your point.

I'll remove the changes from the main README.md

Thanks!

File: archives/2017.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Frontend Conferences for Speaking - 2017
+
+## Conferences
+
+### April
+
+| [JSUnconf](http://jsunconf.eu/) | Hamburg, Germany | April 21 - 22, 2017 | Rolling | Yes | [Yes](http://2018.jsunconf.eu/coc/) |
```
Comment (benmvp) [inline archives/2017.md]: This one is missing the table header which makes it display incorrectly

**Comment by benmvp** [comment]
Thanks for the changes! Sorry for the delay in responding. Was at a conference this weekend. 😄 

Left one fix request for the 2017 archives, then I think it's great to go!

**Comment by benmvp** [comment]
Thanks so much! 🎉

**Pull request merged**